Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -148,6 +149,42 @@ Ref<WebFrame> WebFrame::createSubframe(WebPage& page, WebFrame& parent, const At
     return frame;
 }
 
+Ref<WebFrame> WebFrame::createLocalSubframeHostedInAnotherProcess(WebPage& page, WebFrame& parent, WebCore::FrameIdentifier frameID, WebCore::LayerHostingContextIdentifier layerHostingContextIdentifier)
+{
+    auto frame = create(page);
+    RELEASE_ASSERT(page.corePage());
+    RELEASE_ASSERT(parent.coreAbstractFrame());
```

Comment:
Looks like coreAbstractFrame needs renaming.

Reply:
Will rename in a future PR.  coreFrame is already taken, and renaming that to coreLocalFrame is out of the scope of this PR.